__tests__/src/selectors/thumbnails.test.js

```diff
@@ -0,0 +1,18 @@
+import { Utils } from 'manifesto.js';
+import settings from '../../../src/config/settings';
+import { ThumbnailFactory } from '../../../src/lib/ThumbnailFactory';
+import {
+  getThumbnailFactory,
+} from '../../../src/state/selectors/thumbnails';
+
+/** return the slice of config relevant to MiradorCanvas */
+const miradorConfigSlice = () => ({ auth: settings.auth, canvas: settings.canvas, image: settings.image });
+
+describe('getThumbnailFactory', () => {
+  const state = { config: miradorConfigSlice() };
+  const iiifOpts = {};
+  it('returns the manifest of a certain id', () => {
```

Comment:
Is "the manifest of a certain id" being returned in this test? It seems to expect ThumbnailFactory.

Reply:
You're right. Do you want to suggest a change here, or do you want me to push an additional commit?

Reply:
For some reason I can't suggest a change. Can you push a commit?

Reply:
Done!